tool/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: flutter_tizen_plugin_tools
+description: Productivity utils for flutter_tizen/plugins
+repository: https://github.com/flutter-tizen/plugins/tree/master/tool
+version: 0.1.0
+publish_to: none
+
+environment:
+  sdk: '>=2.15.0 <3.0.0'
+
+dependencies:
+  file: ^6.1.0
+  flutter_plugin_tools: ^0.7.3
```

Comment:
Are you considering expanding `flutter_plugin_tools`?

Reply:
Not expanding but rather reusing some of the common classes already implemented in `flutter_plugin_tools` since many features are not required for our use case.

Reply:
Seems like a good try. thank you for quick answer.